wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/StatementUpdate.java

```diff
@@ -65,8 +72,61 @@ public StatementWithUpdate(Statement statement, boolean write) {
 			this.write = write;
 		}
 	}
+	
+	/**
+	 * Helper class to ease serialization of deleted statements. Jackson will
+	 * serialize this class according to the format required by the API.
+	 * 
+	 * @author antonin
+	 */
+	class DeletedStatement implements Statement {
```

Comment:
Not sure why it is relevant to implement Statement here (but maybe it is)

Reply:
@Tpt it is currently needed, because I push instances of that to a `List<Statement>` later (see https://github.com/Wikidata/Wikidata-Toolkit/pull/275/files#diff-32894a9bb9d9ca9fd81765c38117eb36R185). I found that quite neat but I'm happy to consider other solutions!

Reply:
Ok! Thanks!